packages/compartment-mapper/test-manual/policy/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "endo-sample",
```

Comment:
We try to use `@endo/...` scoping so that those who are watching don't get nervous about NPM hijacking.

```suggestion
  "name": "@endo/compartment-mapper-demo-policy",
```

Reply:
And/or a `scripts.preinstall` that dies loudly.